README.md

```diff
@@ -72,17 +72,11 @@ Ensure that @ symbol is prepended to the file path in the above request. Please
 
 **2. Configure and run your first test**
 
-Open `browserstack_sample.py` file in `Android` or `iOS` folder :
+Open `Android` or `iOS` folder :
 
-- Replace `YOUR_USERNAME` & `YOUR_ACCESS_KEY` with your BrowserStack access credentials. Get your BrowserStack access credentials from [here](https://www.browserstack.com/accounts/settings)
+- If you have uploaded your own app, update the app in the `browserstack.yml`
```

Comment:
Should also add that user can use the path to app directly in the yml file

Reply:
added